src/test/smart-components/portfolio/share-portfolio-modal.test.js

```diff
@@ -194,7 +187,7 @@ describe('<SharePortfolioModal/>', () => {
           initialEntries={['/portfolios/share-portfolio?portfolio=3']}
         >
           <Route
-            path="/portfolios/share-portfolio/"
+            path="/portfolios/share-portfolio"
```

Comment:
And here

Reply:
the ui routes do not have the trailing /